### PR TITLE
Add icon to Add-ons section

### PR DESCRIPTION
**Pull request by yeexinc**
Add an icon to the header of Add-ons section:
addons

The event `addOnsIconClicked` is raised when it is clicked.

@sharadkjaiswal

### CODE REVIEW

**Comment by riteshchandawar** [comment]
The size of an icon looks bigger than the text height, try to keep it same as Text height.

**Comment by yeexinc** [comment]
Thanks @riteshchandawar , here is the updated UI:
addons2

**Comment by yeexinc** [comment]
@sharadkjaiswal - I've made the changes as discussed. Thanks!

File: index.html [hunk 1]
```diff
@@ -56,6 +56,11 @@
                 return null;
             });
 
+            libController.on("sectionIconClicked", function (text) {
```
Comment (sharadkjaiswal) [inline index.html]: Thanks @yeexinc for the update, please merge your PR with #73 and accordingly update the comments and use the constant field used to define the event name.

File: src/components/LibraryItem.tsx [hunk 2]
```diff
@@ -98,6 +98,12 @@ export class LibraryItem extends React.Component<LibraryItemProps, LibraryItemSt
             iconElement = (<img className={"LibraryItemIcon"} src={this.props.data.iconUrl} onError={this.onImageLoadFail} />);
         }
 
+        // If it is a section, display the icon (if given) and provide a click interaction.
+        if (this.props.data.itemType === "section") {
+            iconElement = this.props.data.iconUrl ? (<img className={"LibraryItemIcon"} src={this.props.data.iconUrl} onError={this.onImageLoadFail}
```
Comment (sharadkjaiswal) [inline src/components/LibraryItem.tsx]: What happens on onImageLoadFail() ?

Comment (yeexinc) [inline src/components/LibraryItem.tsx]: The icon will be replaced with this Dynamo icon: https://github.com/DynamoDS/librarie.js/blob/master/src/resources/icons/Dynamo.svg

**Pull request merged**